### PR TITLE
Add an anonymization method to Node.js emitter and API (close #1286)

### LINKED ISSUE #1286: Allow for Node.js anonymization setting
**Is your feature request related to a problem? Please describe.**
Allow for setting anonymization headers on Node.js tracker.

**Describe the solution you'd like**
Have a method to set the anonymization behaviour.

### PR DESCRIPTION
**Pull request by igneel64**
Allow for setting anonymous tracking on the emitter configuration plus on the emitter API using `setAnonymization` .

close #1286

### CODE REVIEW

**Review by adatzer — APPROVED** [review]
Looks good to me!

File: trackers/node-tracker/src/emitter.ts [hunk 1]
```diff
@@ -1,38 +1,9 @@
...
- * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
- * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- */
-
 import { Payload } from '@snowplow/tracker-core';
 
 export interface Emitter {
   flush: () => void;
   input: (payload: Payload) => void;
+  setAnonymization?: (shouldAnonymize: boolean) => void;
```
Comment (matus-tomlein) [inline trackers/node-tracker/src/emitter.ts:6]: Nit: could you add a short docstring here for this function? We are not very good at documenting these types but something is better than nothing...

**Review by matus-tomlein — APPROVED** [review]
LGTM! 👍

File: trackers/node-tracker/src/emitter.ts (hunk 1 above)
Comment (igneel64) [reply trackers/node-tracker/src/emitter.ts:7]: Done!

**Pull request merged**